sonaranalyzer-dotnet/src/SonarAnalyzer.RuleDescriptorGenerator/Program.cs

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
😄

Reply:
Sorry for that. If we squash and merge. It would not appear anymore. Would it?

Reply:
dunno